content/2017-munich/schedule.md

```diff
@@ -4,6 +4,9 @@ title: Schedule
 
 ## PromCon 2017 Schedule
 
+You can also access this schedule in a
+[mobile-friendly format](/2017-munich/mobile_schedule).
```

Comment:
I would change this to "You can also access this schedule via the Giggity mobile app."

Reason is that with the current wording I would simply expect a more mobile-optimized web site and would be disappointed/annoyed to learn only on the next page that I have to install an app (unless of course I want to use an app, in which case it's better to point out that it's an app).